### ISSUE TITLE
Adopt a labeling convention for issues?

### ISSUE THREAD

**Issue by Botmasher** [Botmasher/compchomp#5]
I think issues are easier to understand at a glance if we put some thought into the labels we use to tag them. Adopting [sane GitHub labels](https://medium.com/@dave_lunny/sane-github-labels-c5d2e6004b63) is one example of the process+results of performing this task.
1) Does the default set of labels suffice?
2) If not, what should our set of labels look like?

**Comment by JessRudder** [comment]
Hmmmmm.....the default list looks alright for starters.  But I have a feeling we'll find we're missing some (like labels for script collaboration, perhaps).

There might be some we can get rid of (e.g. "won't fix").

Are there any that you want/want to get rid of after an initial glance?

**Comment by Botmasher** [comment]
I agree about "wontfix". I think adding "documentation" could help, with all the drafts, scripts, texts and logs hanging around.
